(citation)=

# Citing tstrait

If you use `tstrait` in your research project, please cite the following paper:

- Daiki Tagami, Gertjan Bisschop, and Jerome Kelleher (2024), *tstrait: a quantitative trait simulator for ancestral recombination graphs*,Bioinformatics, Volume 40, Issue 6. https://doi.org/10.1093/bioinformatics/btae334

Bibtex records:

```bibtex
@article{10.1093/bioinformatics/btae334,
    author = {Tagami, Daiki and Bisschop, Gertjan and Kelleher, Jerome},
    title = "{tstrait: a quantitative trait simulator for ancestral recombination graphs}",
    journal = {Bioinformatics},
    volume = {40},
    number = {6},
    pages = {btae334},
    year = {2024},
    month = {05},
    abstract = "{Ancestral recombination graphs (ARGs) encode the ensemble of correlated genealogical trees arising from recombination in a compact and efficient structure and are of fundamental importance in population and statistical genetics. Recent breakthroughs have made it possible to simulate and infer ARGs at biobank scale, and there is now intense interest in using ARG-based methods across a broad range of applications, particularly in genome-wide association studies (GWAS). Sophisticated methods exist to simulate ARGs using population genetics models, but there is currently no software to simulate quantitative traits directly from these ARGs. To apply existing quantitative trait simulators users must export genotype data, losing important information about ancestral processes and producing prohibitively large files when applied to the biobank-scale datasets currently of interest in GWAS. We present tstrait, an open-source Python library to simulate quantitative traits on ARGs, and show how this user-friendly software can quickly simulate phenotypes for biobank-scale datasets on a laptop computer.tstrait is available for download on the Python Package Index. Full documentation with examples and workflow templates is available on https://tskit.dev/tstrait/docs/, and the development version is maintained on GitHub (https://github.com/tskit-dev/tstrait).}",
    issn = {1367-4811},
    doi = {10.1093/bioinformatics/btae334},
    url = {https://doi.org/10.1093/bioinformatics/btae334},
    eprint = {https://academic.oup.com/bioinformatics/article-pdf/40/6/btae334/58370902/btae334.pdf},
}
```